### DISCUSSION TITLE
プロジェクト一覧にメモ欄を増やして欲しい

### DISCUSSION THREAD

**Discussion by hiyakake** [vrc-get/vrc-get#1596] [Ideas]
プロジェクト一覧画面に、ユーザーが自由にメモを入力できる欄を増やしていただきたいです。
プロジェクトの管理や、改変状況の整理に役立ちます。

**Comment by anatawa12** [comment]
#375 で仕様について議論しています。UIなどもどこに追加/表示するかなどが良いものが思いついていないので案などがあれば投稿していただけるとありがたいです。
